src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="/images/logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
```

Comment:
as those are on the same level of nesting and there are quite a lot of them - you can add some empty lines between them for better readability

```suggestion
          <li class="nav__item">
            <a href="#Apple" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#Samsung" class="nav__link">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a href="#Smartphones" class="nav__link">Smartphones</a>
          </li>
```